lib/parse.js

```diff
@@ -2109,7 +2109,7 @@ function parse($TEXT, options) {
         return function_(AST_Accessor, is_generator, is_async);
     });
 
-    var object_or_object_destructuring_ = embed_tokens(function() {
+    var object_or_destructuring_ = embed_tokens(function object_or_destructuring_() {
```

Comment:
I don't follow. I'll be honest - I never examined the implementation of `embed_tokens()`. But I thought that function expression was tied exclusively to `var object_or_destructuring_`. Is that not the case?